albam/engines/mtfw/texture.py

```diff
@@ -57,6 +57,19 @@ class TextureType(Enum):  # TODO: TextureTypeSlot
     SPECULAR_BLEND = 22
 
 
+SYMBOL_TEX_TYPES = {
```

Comment:
Instead of creating yet another mapping, wouldn't it better to add these to `TextureType`? 
It shouldn't break any non-re5 code since we don't use the number directly, just the name.

Reply:
I changed the logic in the last commit to use  `TEX_TYPE_MAP_2` dictionary, not sure is it the best idea